applications/vessel-history/src/features/profile/components/Info.tsx

```diff
@@ -201,6 +201,14 @@ const Info: React.FC<InfoProps> = (props): React.ReactElement => {
                   )}
                 </div>
               </div>
+              {vessel.iuuStatus !== undefined && 
```

Comment:
Here the IUU Status will not be shown if we don't have TMT data, that's not correct. Because if we don’t have TMT data, IUU listed status should display `unknown`